students/eowyn/session06/mailroom/mailroom.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+
+import sys
+
+''' --------------------------------------------------------------
+GLOBAL VARIABLE: DONORS is the database of donor names, amounts
+-------------------------------------------------------------------'''
+donor_names = ["Margaret Atwood", "Fred Armisen",
+               "Heinz the Baron Krauss von Espy"]
+donations = [[300, 555], [240, 422, 1000], [1500, 2300]]
+DONORS = dict(zip(donor_names, donations))
+del donor_names
+del donations
+
+
+''' --------------------------------------------------------------
+Following are helper functions to control program flow
+-------------------------------------------------------------------'''
+
+
+def safe_input():
+    return None
+
+
+def quit_code():
+    sys.exit()
+
+
+def return_to_menu():
+    ''' Return True to trigger exit out of sub-loop'''
+    return True
+
+
+''' --------------------------------------------------------------
```

Comment:
you want to use comments, rather than triple-quoted strings for this sort of thing:

```
# Following are helper functions for generating reports
# -----------------------------------------------------
```